model/task/task.go

```diff
@@ -2741,6 +2743,35 @@ func GetTasksByVersion(versionID string, sortBy []TasksSortOrder, statuses []str
 					tempParentKey: []interface{}{},
 				},
 			},
+			// get any annotation that has at least one issue
+			{
+				"$lookup": bson.M{
+					"from": annotations.Collection,
+					"let":  bson.M{"task_annotation_id": "$" + IdKey, "task_annotation_execution": "$" + ExecutionKey},
+					"pipeline": []bson.M{
+						{
+							"$match": bson.M{
+								"$expr": bson.M{
+									"$and": []bson.M{
+										{
+											"$eq": []string{"$" + annotations.TaskIdKey, "$$task_annotation_id"},
+										},
+										{
+											"$eq": []string{"$" + annotations.TaskExecutionKey, "$$task_annotation_execution"},
+										},
+										{
+											"$ne": []interface{}{
+												bson.M{
+													"$size": bson.M{"$ifNull": []interface{}{"$" + annotations.IssuesKey, []bson.M{}}},
+												}, 0,
+											},
+										},
+									},
+								},
+							}}},
+					"as": "annotation_doc",
```

Comment:
nit: can we call this something plural? Doesn't really matter but will make it more clear what's happening here

Reply:
done